config/routes.js

```diff
@@ -66,6 +66,7 @@ module.exports.routes = {
   'GET    /noo/community/:communityId':                   'CommunityController.findOne',
   'POST   /noo/community/:communityId':                   'CommunityController.update',
   'GET    /noo/community/:communityId/settings':          'CommunityController.findSettings',
+  'GET    /noo/community/:communityId/settings/slack':    'CommunityController.addSlack',
```

Comment:
My main concern is that a GET with no parameters is often a good candidate for HTTP caching, either by the browser or by any sort of proxy in between (at ISP's, etc). So using a POST is the right thing to do because it signals never to use a cache.